samples/sensor/soc_voltage/README.rst

```diff
@@ -0,0 +1,30 @@
+.. _stm32_vref_sensor:
```

Comment:
Need to update this anchor

Reply:
Good catch! Missed the README with my replace regex it seems when updating the sample.